extensions/eclipse-che-theia-terminal/src/common/workspace-service.ts

```diff
@@ -14,6 +14,9 @@ export const cheWorkspaceServicePath = '/services/che-workspace-service';
 
 export const CHEWorkspaceService = Symbol('CHEWorkspaceService');
 
+export const RECIPE_CONTAINER_SOURCE = 'recipe';
+export const CONTAINER_SOURCE_ATTRIBUTE = 'source';
+
```

Comment:
Looks like you added constants here, but you don't use them: you use the constants from `terminal-command-filter`.

Reply:
Added by mistake. Fixed